### PR TITLE
CNV-7169 RN for Golden Images

### PR DESCRIPTION
**Pull request by bgaydosrh**
First draft of Release Note for Golden Image stories [CNV-5259](https://issues.redhat.com/browse/CNV-5259) and [CNV-7193](https://issues.redhat.com/browse/CNV-7193):

https://issues.redhat.com/browse/CNV-5259
https://issues.redhat.com/browse/CNV-7193

Draft text of note:

_You can now create default OS images and automatically upload them using the {product-title} Web Console. A _default OS image_ is a bootable disk containing an operating system and all of the operating system's configuration settings, such as drivers. You use a default OS image to create bootable virtual machines with specific configurations._

Tagging @aglitke for code review, will add Tomas in Jira.
Tagging @demilyc for QE review.

Thanks
Bob

### CODE REVIEW

**Review by aglitke — COMMENTED** [review]
looks good to me.

**Comment by jelkosz** [comment]
lgtm

File: virt/virt-2-5-release-notes.adoc [hunk 1]
```diff
@@ -101,6 +101,7 @@ The *Pending Changes* banner at the top of the page displays a list of all chang
 //CNV-7318 - Open the vnc/serial console in a new window
 
 //CNV-7169 - Auto-clone base images when creating VM in UI
+You can now create default OS images and automatically upload them using the {product-title} Web Console. A _default OS image_ is a bootable disk containing an operating system and all of the operating system's configuration settings, such as drivers. You use a default OS image to create bootable virtual machines with specific configurations.
```
Comment (adellape) [inline virt/virt-2-5-release-notes.adoc:104]: It looks like this should be a bulleted item, to match the ones around it.

Comment (adellape) [reply virt/virt-2-5-release-notes.adoc:104]: Also:

s/Web Console/web console/

To match usage elsewhere in the repo.

**Review by adellape — COMMENTED** [review]
Small nits.

File: virt/virt-2-5-release-notes.adoc (hunk 1 above)
Comment (bgaydosrh) [reply virt/virt-2-5-release-notes.adoc]: Done.

**Pull request merged**